src/ripple/site_scons/scons_to_ninja.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2014 The Native Client Authors. All rights reserved.
```

Comment:
The formatting here is near pathological - not _one_ blank line?

(I know you didn't write this, but this is very non-PEP-8.)